cmd/gcp-controller-manager/main.go

```diff
@@ -72,7 +72,6 @@ var (
 	authSyncNodeURL                       = pflag.String("auth-sync-node-url", "", "URL for reaching the Auth Service SyncNode API.")
 	hmsAuthorizeSAMappingURL              = pflag.String("hms-authorize-sa-mapping-url", "", "URL for reaching the Hosted Master Service AuthorizeSAMapping API.")
 	hmsSyncNodeURL                        = pflag.String("hms-sync-node-url", "", "URL for reaching the Hosted Master Service SyncNode API.")
-	autopilotEnabled                      = pflag.Bool("autopilot", false, "Is this a GKE Autopilot cluster.")
```

Comment:
if you remove flags, independently they maybe a noop, the manifests using this binary need to be updated or will fail to execute